www/front_src/src/components/Wallpaper/BackgroundImage.tsx

```diff
@@ -34,7 +32,7 @@ const BackgroundImage = ({ image }: Props): JSX.Element => {
 
   return (
     <Fade in={not(isNil(image))}>
-      <div aria-label={labelCentreonWallpaper} className={classes.wallpaper} />
+      <div className={classes.wallpaper} id="Centreon_wallpaper" />
```

Comment:
@Thebarda What do you think about this syntax ?
```suggestion
      <div className={classes.wallpaper} id="centreon-wallpaper" />
```